src/server/object_services/md_store.js

```diff
@@ -906,6 +906,23 @@ class MDStore {
         });
     }
 
+    /**
+     * @param {nb.ID} obj_id
+     * @returns {Promise<nb.ObjectMultipart[]>}
+     */
+    async find_commited_multiparts_of_object(obj_id) {
+        return this._multiparts.find({ obj: obj_id, deleted: null, uncommitted: null });
```

Comment:
We should not forget that ```uncommitted``` is not indexed
Just commenting out since I can see that this query is added on this property in addition to pre-existing queries: ```find_parts_by_start_range``` and ```find_parts_sorted_by_start```

Reply:
I think we should think about adjusting the indexes, but not as part of this PR

Reply:
Why are we calling find and not count? Is there any need to read those multiparts to memory?